server/controllers/settings.js

```diff
@@ -5,6 +5,8 @@ import {ADMIN_ROLE, volunteerRoles} from '../../common/constants'
 import config from '../config'
 import Settings from '../models/settings'
 
+import {updateLocationSettings} from '../lib/settings-location-update'
```

Comment:
Why can't this be a pre-save method too? Otherwise I'd just make it a function in this file to take the address and return the location, then add the location to the request body before saving. Then you don't need to lookup and save twice.